lib/Service/VerdictService.php

```diff
@@ -141,6 +155,27 @@ public function scan(string $filePath): VaasVerdict {
 		}
 	}
 
```

Comment:
In order to adapt to the previous conditions of the app the functions should all have PHPDocs